### DISCUSSION TITLE
Do we will have a HarmonyOS  version？

### DISCUSSION THREAD

**Discussion by HAXIOUS** [immich-app/immich#11307] [General]
New HarmonyOS NEXT didnt support Android APP any more. I think we need a HarmonyOS version,even it need to be paid or only has simplest functions like upload photos.

**Comment by bo0tzz** [comment]
It's unlikely that the core team will spend time on that, but if flutter can be built for harmonyos then anybody is free to create a build.

**Reply by HAXIOUS** [reply]
OK,thank you very much

**Reply by baozi510** [reply]
我也在发愁这个问题，我不打算升级，但是怕强制给我升级。
